snippets/comparison/client and server components.md

```diff
@@ -63,3 +67,5 @@ export default function Page() {
   );
 }
 ```
+
+You can read more about Server and Client Components in [here](https://beta.nextjs.org/docs/rendering/server-and-client-components).
```

Comment:
Prefer self descriptive links instead of `here`. Prefer something like:

```suggestion
You can read more about Server and Client Components in the [official documentation](https://beta.nextjs.org/docs/rendering/server-and-client-components).
```